src/mobile/apps/partner_profile/templates/contact.jade

```diff
@@ -41,4 +41,7 @@ block content
               p: a.avant-garde-box-button( href="tel:#{location.get('phone')}" ) Call #{partner.get('type')}
             br
 
+    if partner.has('vat_number')
+      .partner-vat-info
+        | VAT ID #: #{partner.get('vat_number')}
```

Comment:
Do we also want to use `unica` for mobile display?

Reply:
We just chatted about this IRL - since we don't currently use Unica anywhere in `/mobile`, we're fine to leave it with Garamond